setup.py

```diff
@@ -53,6 +53,10 @@ def read_version(file_path="version.txt"):
     and platform.system() == "Darwin"
 )
 
+use_cpp_avx512 = os.getenv("USE_AVX512", "1") == "1" and platform.system() == "Linux"
```

Comment:
Can we use PyTorch APIs such as `torch._C._cpu._is_avx512_supported`? Thanks!

Reply:
Thanks for the suggestion. It works!

Reply:
Update: The API is new, so we would encounter error for torch with lower versions.